indexes.json

```diff
@@ -453,6 +453,13 @@
       "category": "Component",
       "description": "combine computed and inject"
     },
+    {
+      "name": "loadify",
+      "package": "core",
```

Comment:
`indexes.json` file now is in `.gitignore`. Delete this from your pr to pass tests. I had the same problem.